src/testrender/optixraytracer.cpp

```diff
@@ -527,20 +529,18 @@ OptixRaytracer::make_optix_materials ()
     size_t sizeof_msg_log;
 
     // Make module that contains programs we'll use in this scene
-    OptixModuleCompileOptions module_compile_options;
+    OptixModuleCompileOptions module_compile_options = {};
```

Comment:
Can I ask... what is going on with this line?

Reply:
Just making sure the options are zero-initialized.